Client/Frontend/Home/Wallpapers/WallpaperDataManager.swift

```diff
@@ -4,10 +4,17 @@
 
 import Foundation
 
+/// An internal struct used to define wallpaper names and their respective
+/// accessibility strings for voice-over.
+private struct WallpaperID {
+    let name: String
+    let accessibilityID: String
```

Comment:
This looks like the accessibility label and not the identifier (looking at how this is used). Label is what is read out loud to the user.